statsmodels/base/_penalized.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May 10 08:23:48 2015
```

Comment:
timestamp accurate?

Reply:
Yes, that was timestamp in my original PR